apps/web/package.json

```diff
@@ -12,7 +12,7 @@
     "test-codegen": "yarn playwright codegen http://localhost:3000",
     "type-check": "tsc --pretty --noEmit",
     "type-check:ci": "tsc-absolute --pretty --noEmit",
-    "build": "next build",
+    "build": "NODE_OPTIONS=\"--max-old-space-size=8192\" next build",
```

Comment:
we added this here for now because configuring `NODE_OPTIONS` in `production-build.yml` didn't work somehow. Can you advise us on why that would be the case?